ios/Classes/SwiftBccmPlayerPlugin.swift

```diff
@@ -39,4 +53,8 @@ public class SwiftBccmPlayerPlugin: NSObject, FlutterPlugin {
     public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
         result("iOS " + UIDevice.current.systemVersion)
     }
+    
+    public func application(_ application: UIApplication, handleEventsForBackgroundURLSession identifier: String, completionHandler: @escaping () -> Void) -> Bool {
+        return true
```

Comment:
Not quite sure what this does but I'd assume you would need to at least check the "identifier" here?

Returning true means "yup we handled this one" so it seems scary in case some other plugin needs to handle it.

https://api.flutter.dev/objcdoc/Protocols/FlutterApplicationLifeCycleDelegate.html#/c:objc(pl)FlutterApplicationLifeCycleDelegate(im)application:handleEventsForBackgroundURLSession:completionHandler: